crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

```diff
@@ -9,6 +9,142 @@ Changelog - nrf_cc3xx_mbedcrypto
+
+* Bugfixes and improvements.
+* Improved parameter-testing for invalid or empty inputs/outputs.
+* Fixed PSA Crypto driver APIs for AES CCM so it supports multiple calls to add AAD.
+* Fixed PSA Crypto driver APIs for ECDH using Montgomery curves so they support 255 bit curves (was 256 bits) 
+* Corrected invalid return-codes being reported for some PSA crypto driver APIs.
+* Changed the API for PSA Cipher for nrf_cc3xx PSA Crypto driver (now includes iv and iv_length parameters).
+
+Library built against Mbed TLS version 3.1.0.
+
+This version is dependent on the nrf_cc310_platform or nrf_cc312_platform library for low-level initialization of the system and proper RTOS integration.
+
+Added
+=====
+
+Added a new build of nRF_cc3xx_mbedcrypto libraries for nRF9160, nRF52840, and nRF5340.
+
+.. note::
+
+   The *short-wchar* libraries are compiled with a wchar_t size of 16 bits.
+
+* nrf_cc312_mbedcrypto, nRF5340 variants
+
+  * ``cortex-m33/hard-float/libnrf_cc312_psa_crypto_0.9.15.a``
+  * ``cortex-m33/hard-float/libnrf_cc312_legacy_crypto_0.9.15.a``
+  * ``cortex-m33/hard-float/libnrf_cc312_core_0.9.15.a``
+  * ``cortex-m33/soft-float/libnrf_cc312_psa_crypto_0.9.15.a``
+  * ``cortex-m33/soft-float/libnrf_cc312_legacy_crypto_0.9.15.a``
+  * ``cortex-m33/soft-float/libnrf_cc312_core_0.9.15.a``
+
+  * No interrupts
+
+    * ``cortex-m33/soft-float/no-interrupts/libnrf_cc312_psa_crypto_0.9.15.a``
+    * ``cortex-m33/soft-float/no-interrupts/libnrf_cc312_legacy_crypto_0.9.15.a``
+    * ``cortex-m33/soft-float/no-interrupts/libnrf_cc312_core_0.9.15.a``
+    * ``cortex-m33/hard-float/no-interrupts/libnrf_cc312_psa_crypto_0.9.15.a``
+    * ``cortex-m33/hard-float/no-interrupts/libnrf_cc312_legacy_crypto_0.9.15.a``
+    * ``cortex-m33/hard-float/no-interrupts/libnrf_cc312_core_0.9.15.a``
+
+  * short-wchar
+
+    * ``cortex-m33/hard-float/short-wchar/libnrf_cc312_psa_crypto_0.9.15.a``
+    * ``cortex-m33/hard-float/short-wchar/libnrf_cc312_legacy_crypto_0.9.15.a``
+    * ``cortex-m33/hard-float/short-wchar/libnrf_cc312_core_0.9.15.a``
+    * ``cortex-m33/soft-float/short-wchar/libnrf_cc312_psa_crypto_0.9.15.a``
+    * ``cortex-m33/soft-float/short-wchar/libnrf_cc312_legacy_crypto_0.9.15.a``
+    * ``cortex-m33/soft-float/short-wchar/libnrf_cc312_core_0.9.15.a``
+
+  * short-wchar, no interrupts
+
+    * ``cortex-m33/hard-float/short-wchar/no-interrupts/libnrf_cc312_psa_crypto_0.9.15.a``
+    * ``cortex-m33/hard-float/short-wchar/no-interrupts/libnrf_cc312_legacy_crypto_0.9.15.a``
+    * ``cortex-m33/hard-float/short-wchar/no-interrupts/libnrf_cc312_core_0.9.15.a``
+    * ``cortex-m33/soft-float/short-wchar/no-interrupts/libnrf_cc312_psa_crypto_0.9.15.a``
+    * ``cortex-m33/soft-float/short-wchar/no-interrupts/libnrf_cc312_legacy_crypto_0.9.15.a``
+    * ``cortex-m33/soft-float/short-wchar/no-interrupts/libnrf_cc312_core_0.9.15.a``
+
+
+* nrf_cc310_mbedcrypto, nRF9160 variants
+
+  * ``cortex-m33/hard-float/libnrf_cc312_psa_crypto_0.9.15.a``
```

Comment:
```suggestion
  * ``cortex-m33/hard-float/libnrf_cc310_psa_crypto_0.9.15.a``
```

Either fix this or delete this.

It's not clear to me that this should be in the changelog at all.

Reply:
e.g. we could avoid copy-paste errors and reduce paperwork without hurting the docs (IMHO) by replacing all this noise with:

nrf_cc310_mbedcrypto, nRF9160 variants

* crypto/nrf_cc310_mbedcrypto/lib/cortex-m33/**/libnrf_cc310_core_0.9.15.a

nrf_cc312_mbedcrypto, nRF5340 variants

* crypto/nrf_cc312_mbedcrypto/lib/cortex-m33/**/libnrf_cc312_core_0.9.15.a

Reply:
oh, and the same for 52

Reply:
I used this approach